client/src/locales/en-US.json

```diff
@@ -1625,7 +1625,28 @@
                 "description": "You can change your credit card at any time.",
                 "update": "Update"
             },
-            "processing": "Your order is being processed."
+            "processing": {
+                "requestSent": {
+                    "title": "Request sent",
+                    "description": "Your request is being processed. We will contact you within 4 working days."
+                },
+                "estimateLinked": {
+                    "title": "Quotation signature pending",
+                    "description": "Please sign and send back the quotation received by e-mail to get your invoice."
```

Comment:
Estimates are not forced to be signed, a customer could even only have directly an invoice, so this sentence needs to be changed.